wire-ios-data-model/Source/ManagedObjectContext/Migration/2.110-2.111/PrefillPrimaryKeyAction.swift

```diff
@@ -38,11 +38,15 @@ class PrefillPrimaryKeyAction: CoreDataMigrationAction {
             let request = NSFetchRequest<NSManagedObject>(entityName: entityName)
             request.fetchBatchSize = batchSize
             let objects = try context.fetch(request)
-
+            var index = 0
             objects.forEach { object in
```

Comment:
Maybe we can add a log how many objects will be updated to make this step visible in the happy case.